packages/theming/android-theme/src/androidTypography.ts

```diff
@@ -17,33 +17,35 @@ export function androidTypography(): Typography {
       heroLarge: 28 as FontSize, // Headline
     } as FontSizes,
     weights: {
-      light: '300' as FontWeightValue,
+      // The font weights do not work for Android since RN 0.60 (https://github.com/facebook/react-native/issues/25696)
+      // The workaround is to use font aliases from android source (https://android.googlesource.com/platform/frameworks/base/+/master/data/fonts/fonts.xml)
       regular: '400' as FontWeightValue,
-      medium: '500' as FontWeightValue,
       semiBold: '500' as FontWeightValue,
     },
     families: {
-      primary: 'Roboto',
+      primary: 'sans-serif', // sans-serif, weight=400
+      primaryBold: 'sans-serif-medium', // sans-serif, weight=500
+      primaryLight: 'sans-serif-light', // sans-serif, weight=300
       secondary: 'System',
       cursive: 'System',
       monospace: 'monospace',
       sansSerif: 'sans-serif',
       serif: 'serif',
     },
```

Comment:
Is it OK for themes to define their own font families? I would assume so but I would worry that would get confusing for clients writing x-plat code unless we explicitly documented it somewhere, like the in the README.md of the Android Theme package.

Reply:
This is probably a good question to discuss with Travis when we start hooking up to the pipeline, and agree on an answer for this.

Reply:
@E.J.Layne was there anymore discussion here? I'm not really sure what else could be done. We had to do something similar for win32 where we needed to define a font face for semibold variants (since the font weight didn't work) and I stuck it under secondary instead of defining a new font family but perhaps I should have named it primarySemibold like it's done here.